src/contexts/LocationContext.js

```diff
@@ -0,0 +1,21 @@
+import React, { useState, createContext } from 'react';
+
+export const LocationContext = createContext();
+
+const LocationContextProvider = (props) => {
+    let locStorage = localStorage.getItem("locationValue");
+    const [currLocation, setCurrLocation] = useState((locStorage) ? locStorage : '/timeline');
+
+    const getLocation = (loc) => {
```

Comment:
this method is more like `setLocation`?

Reply:
you're right... setLoc is more appropriate, any changes from this will be commited to latest branch (pull#28)